modules/apps/commerce/commerce-product-asset-categories-web/src/main/resources/META-INF/resources/js/EditAssetCategoryCPDisplayLayout.es.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+alert('yay');
```

Comment:
> Failed venture in moving the JS into its own file - for now!

Well, this line, at least, is a success!

![rocky](https://user-images.githubusercontent.com/7074/117304484-15d00a00-ae7e-11eb-869d-3e6bc3686418.gif)

Reply:
`alert('yay');`
![yay](https://media0.giphy.com/media/yoJC2COHSxjIqadyZW/giphy.gif)

Reply:
![](https://media2.giphy.com/media/G1vplGMypxBcp7kx32/giphy.gif)